package.json

```diff
@@ -33,7 +33,9 @@
         "static-p": "yarn build && yarn clean-static && yarn collectstatic-p",
         "watch-front-p": "yarn front yarn watch",
         "start-back-p": "cd backend && python manage.py runserver --settings production --insecure --pythonpath ..",
-        "start-p": "yarn static-p && yarn watch-front-p & yarn start-back-p"
+        "start-p": "yarn static-p && yarn watch-front-p & yarn start-back-p",
+        "stop-back": "kill $(lsof -t -i:8000)",
+        "codegen-once": "yarn start-back & yarn front yarn graphql-codegen && yarn stop-back"
```

Comment:
On my system, running `yarn codegen-once` simply starts the backend server without (visibly) running codegen or quitting. Maybe this is because `lsof` does not work on Windows. I'm happy to leave it as is, but the documentation should probably mention that Windows users are out of luck in this respect :P